platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -792,7 +799,17 @@ def get_firmware_update_notification(self, image_path):
         return "Immediate power cycle is required to complete {} firmware update".format(self.name)
 
     def install_firmware(self, image_path):
-        return self.__install_firmware(image_path)
+        if ".mpfa" in image_path:
```

Comment:
@alexrallen suggest having a constant for `.mpfa`

Reply:
Agree